src/Homework1.java

```diff
@@ -1,101 +1,272 @@
 public class Homework1 {
 
     public static void main(String[] args) {
-        // Primitives
-        int myPrimitive = 10;
-        System.out.println(Integer.toBinaryString(myPrimitive));
-        System.out.println(Integer.toBinaryString(Integer.MAX_VALUE));
-
-        float myFloatPrimitive = 10.5f;
-        double myDoublePrimitive = 10.5f;
-        System.out.println(myFloatPrimitive);
-
-        char myCharPrimitive = 'c';
-
-        boolean myBooleanPrimitive = true;
-        boolean myFalseBooleanPrimitive = false;
-
-        // Objects
-        Integer myIntegerPrimitive = 10;
-
-        String myString = "Hello PentaStagiu!";
-
-
-        // Control blocks
-
-        // Display parity
-        int myVar = 11;
-        if (myVar % 2 == 0) {
-            System.out.println("The variable is even!");
-        } else {
-            System.out.println("The variable is odd!");
-        }
-
-//        if (myVar % 2 != 0) {
-//            System.out.println("The variable is odd!");
-//        }
-
-        // Display week-days
-        byte weekDay = 50;
-
-        switch (weekDay) {
-            case 1:
-                System.out.println("Mon");
+        int problem = 1;
+        switch (problem) {
+            case 1: Problem1();
+                    break;
+            case 2: Problem2();
+                break;
+            case 3: Problem3();
+                break;
+            case 4: Problem4();
                 break;
-            case 2:
-                System.out.println("Tue");
+            case 5: Problem5();
                 break;
-            case 3:
-                System.out.println("Wen");
+            case 6: Problem6();
                 break;
-            case 4:
-                System.out.println("Thu");
+            case 7: Problem7();
                 break;
-            case 5:
-                System.out.println("Fri");
+            case 8: Problem8();
                 break;
-            case 6:
-                System.out.println("Sat");
+            case 9: Problem9();
                 break;
-            case 7:
-                System.out.println("Sun");
+            case 10: Problem10();
+                break;
+            case 11: Problem11();
+                break;
+            case 12: Problem12();
+                break;
+            case 13: Problem13();
+                break;
+            case 14: Problem14();
+                break;
+            case 15: Problem15();
                 break;
             default:
-                System.out.println("Wrong number: " + weekDay);
+                System.out.println("Wrong problem");
         }
+    }
 
+    public static void Problem1()
+    {
+        int inputNumber = 5;
+        int sum = 0;
+        for(int i = 1; i<=inputNumber; ++i){
```

Comment:
Este bine doar ca in enunt spune sa se faca suma de la 0 la inputNumber, deci ar fi trebuit ca i-ul sa plece de la 0. Dar nu e nicio problema, doar ca de obicei se pleaca de la 0 deoarece indexul array-urilor pleaca tot de la 0.

Reply:
Solved.